roles/bionic/postgresql10/defaults/main.yml

```diff
@@ -0,0 +1,5 @@
+---
+postgresql_version: 10
+postgresql_apt_key_id: ACCC4CF8
+postgis_version: 2.4
```

Comment:
What is this key for? `postgis_enabled`

Reply:
I plan to set the option for install only postgres or install with extension